server/src/test/java/org/elasticsearch/action/admin/indices/datastream/GetDataStreamsRequestTests.java

```diff
@@ -65,8 +60,9 @@ public void testGetDataStreams() {
     public void testGetNonexistentDataStream() {
```

Comment:
I think we should add a bit more tests:

1. Get non existent data stream based on wildcard (which should return empty array).
2. Get data stream based on wildcard, returning multiple.